docs/control-structures.rst

```diff
@@ -2,6 +2,56 @@
 Expressions and Control Structures
 ##################################
 
+.. index:: ! parameter, parameter;input, parameter;output
+
+Input Parameters and Output Parameters
+======================================
+
+Like in Javascript and in C, functions may take parameters as input;
```

Comment:
I wonder which languages do we need to compare Solidity to. The only one mentioned in the documentation so far is Javascript in the index:
> Solidity is a contract-oriented, high-level language whose syntax is similar to that of JavaScript and it is designed to target the Ethereum Virtual Machine.

Should we compare it to Javascript or just say it is a "curly bracket language"? :)